app/helpers/blacklight_helper.rb

```diff
@@ -9,23 +9,37 @@ def render_search_bar
     render partial: 'catalog/franklin_search_bar'
   end
 
-  def render_expert_help(specialists)
-    if specialists.blank? || specialists.items.first.hits < 50_000
+  def render_expert_help(specialists_facet_field)
+    return render partial: 'catalog/ask' unless specialists_facet_field
+    specialists = extract_and_sort_by_relatedness(specialists_facet_field)
+    if specialists.blank? || specialists.first.subs[:r1][:relatedness] < 0.3
```

Comment:
this `0.3` seems like a value that can be subject to "tuning" - as such perhaps it should be defined as a constant?